Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -1040,6 +1040,11 @@ bool CLuaPedDefs::UpdateElementRpHAnim(lua_State* const luaVM, CClientEntity* en
     if (theEntity)
     {
         theEntity->UpdateRpHAnim();
+        if (theEntity->GetModelIndex() == 0) { // CJ skin
+            RpClump* clump = theEntity->GetRpClump();
+            if (clump)
+                ((void(__cdecl*)(RpClump*))0x5DF560)(clump); // CPed::ShoulderBoneRotation
```

Comment:
I think blocks without braces should be generally avoided.

Reply:
you're right, thank you